.pre-commit-config.yaml

```
@@ -106,3 +106,12 @@ repos:
         - Babel==2.12.1
         - multipart==0.2.4
         - "git+https://github.com/webpy/webpy.git@d3649322b85777b291ac2b7b3699fb6fc839e382"
+
+  - repo: local
+    hooks:
+    - id: detect-missing-i18n
+      name: Detect missing i18n
+      description: This hook checks .html files for i18n strings that might be missing.
+      entry: time python ./scripts/detect_missing_i18n.py
```

Ah seems like we need to drop `time` here to make the github CI happy! https://results.pre-commit.ci/run/github/69609/1718565101.yL9bSiQcSAqNRXAwORPvPw Committing this fix to your branch real quick so I can see the output via GH :+1: 

```suggestion
      entry: python ./scripts/detect_missing_i18n.py
```